monai/transforms/croppad/functional.py

```diff
@@ -148,23 +149,26 @@ def crop_or_pad_nd(img: torch.Tensor, translation_mat, spatial_size: tuple[int,
 
 
 def pad_func(
-    img: torch.Tensor, to_pad: tuple[tuple[int, int]], mode: str, transform_info: dict, kwargs
+    img: torch.Tensor, to_pad: tuple[tuple[int, int]], transform_info: dict, mode: str=PytorchPadMode.CONSTANT, **kwargs
```

Comment:
it seems that this change is not needed by this PR, could you please confirm?

Reply:
Since the last line here https://github.com/Project-MONAI/MONAI/blob/be3d13869d9e0060d17a794d97d528d4e4dcc1fc/monai/transforms/croppad/functional.py#L150-L164 mentioned the default padding mode is `PytorchPadMode.CONSTANT` (consistent with `pad_nd`'s signature), which make `mode` become a keyword parameter, the order of parameters needs to be changed. Would we keep this change or use the old `pad_func`'s signature?

Reply:
I see, thanks, I think we can keep this change, also `mode` and `kwargs` are logically related, makes sense.